### PR TITLE
Add Axolstudio Helpo

### PR DESCRIPTION
**Pull request by kb-elmo**
## Description

Added the Axolstudio Helpo Macropad (soon to be open sourced)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — APPROVED** [review]

File: keyboards/axolstudio/helpo/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        KC_1,    KC_2,    KC_3,    KC_4,    KC_5, \
+        KC_6,    KC_7,    KC_8,    KC_9,    KC_0, \
+        KC_A,    KC_B,    KC_C,    KC_D,    KC_E, \
+        KC_F,    KC_G,    KC_H,    KC_I,    MO(1) \
+    ),
+    [1] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS  \
```
Comment (zvecr) [inline keyboards/axolstudio/helpo/keymaps/default/keymap.c:30]:
```suggestion
        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,
        KC_6,    KC_7,    KC_8,    KC_9,    KC_0,
        KC_A,    KC_B,    KC_C,    KC_D,    KC_E,
        KC_F,    KC_G,    KC_H,    KC_I,    MO(1)
    ),
    [1] = LAYOUT(
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
```

File: keyboards/axolstudio/helpo/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS  \
+    ),
+    [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS  \
+    )
```
Comment (zvecr) [inline keyboards/axolstudio/helpo/keymaps/via/keymap.c:43]:
```suggestion
    [0] = LAYOUT(
        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,
        KC_6,    KC_7,    KC_8,    KC_9,    KC_0,
        KC_A,    KC_B,    KC_C,    KC_D,    KC_E,
        KC_F,    KC_G,    KC_H,    KC_I,    MO(1)
    ),
    [1] = LAYOUT(
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
    ),
    [2] = LAYOUT(
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
    ),
    [3] = LAYOUT(
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
    )
```

**Review by zvecr — CHANGES_REQUESTED** [review]
Forgot to check for these

**Review by drashna — APPROVED** [review]

File: keyboards/axolstudio/helpo/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Helpo
+
+*ATmega32A powered 20 key THT macropad*
```
Comment (fauxpark) [inline keyboards/axolstudio/helpo/readme.md:3]:
```suggestion
ATmega32A powered 20 key THT macropad
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**